### DISCUSSION TITLE
Does this adapter support POE Ethernet to usb for the Wyze Cam v3? - UCTRONICS IEEE 802.3af Micro USB Active PoE Splitter

### DISCUSSION THREAD

**Discussion by mavacpjm** [gtxaspec/wz_mini_hacks#784] [Q&A]
Before I got testing, does this adapter work with wz_mini-hacks?

UCTRONICS IEEE 802.3af Micro USB Active PoE Splitter Power Over Ethernet 48V to 5V 2.4A for Tablets, Dropcam or Raspberry Pi 2/3B+ (48V to 5V 2.4A) 

https://www.amazon.com/gp/product/B01MDLUSE7/ref=ox_sc_act_title_1?smid=A2IAB2RW3LLT8D&psc=1

Just want to be sure its compatible.... TIA !!!

**Comment by Bob-AOL** [comment]
Depending on your application, this one works great for a straight PoE connection to a V3. 

https://www.amazon.com/dp/B0876N3PMZ?ref=emc_s_m_5_i_atc

Quite a bit of discussion about it in various locations here.

**Reply by mavacpjm** [reply]
that specific one does not work, you have to use the upgraded uctronics or it will not work. here is the link to the upgraded one.

https://www.amazon.com/gp/product/B0B9ZLVGJ4/ref=ox_sc_act_title_1?smid=A2IAB2RW3LLT8D&psc=1

**Reply by Bob-AOL** [reply]
I originally bought three of the ones I listed and they all worked just fine. Granted, there were problems with that version initially which prompted the 'improved version'. It had its own problems and was the same basic crap shoot until they got those sorted out as well.

That said, and knowing how stock rotation/distribution goes (or doesn't) with Amazon and its vendors, I would probably opt for the one you listed just because -- and you could still very well receive one of the earlier ones anyway.

Bottom line is they work great when they work right and are fixable if you still get a buggy one. Trust me, I wrote the book on these things.

**Comment by tismofied** [comment]
Is there any budget friendly ones that work? Even from Ali express? 
Those are $20 a pop and for 5 cameras that's a cool $100 just for the adaptors.

**Comment by Bob-AOL** [comment]
Buy em one at a time and it won't hurt as much. 

They turn a 30 dollar hobby cam into a PoE IP camera for another 20 bucks.

You either find value in that or you don't.

**Reply by tismofied** [reply]
That's fair.
